content/1.7/back-office/shipping/carriers/Add-edit-carrier.md

```diff
@@ -0,0 +1,143 @@
+## Columns
+
+Firstly on the left the logo is displayed. Whena  logo is uploaded, a trash can icon appears on the right of the logo. When trying to delete the logo, message appears: "Are you sure you want to delete the logo?". Pressing "Ok" will remove the uploaded logo, and the default van logo will re-appear in the same place.
+
+On the right side, from left to the right, the steps of adding/editing a carrier are displayed: 
+
+ - 1 General settings
+ - 2 Multistore (If multistore is enabled)
+ - 3 Shipping locations and costs
+ - 4 Size, weight, and group access
+ - 5 Summary
+
+Below there 5 fields:
+
+ - Carrier name. This is a mandatory field, when trying to proceed to the next step without inputting this field, shows the message: "The name field is required." When trying to enter invalid symbols to this field, and trying to proceed to the next step, will show the message: "The name field is invalid".
+A tooltip is displayed: "Allowed characters: letters, spaces and "().-". The carrier's name will be displayed during checkout. For in-store pickup, enter 0 to replace the carrier name with your shop name."
+
+ - Transit time. This field is mandatory, when trying to proceed to the next step without inputting this field, shows the message: "The field delay is required at least in English (English).". This field can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+A tooltip is displayed: "The delivery time will be displayed during checkout.".
+
+ - Speed grade. Only one number from 0 to 9 is allowed in this field. Entering an incorrect symbol, shows the message: "The grade field is invalid.". Entering more than 1 symbol in the field will show the following: "The grade field is too long (1 chars max).".
+A tooltip is displayed: "Ënter "0" for a longest shipping delay, or "9" for the shortest shipping delay"
+
+ - Logo. A field for uploading the logo. To upload a logo, there is a button on the right of the field named ""Choose a file". Has an explanation below the field: "Format: JPG, GIF, PNG. Filesize: 8.00 MB max. Current size: undefined.". When trying to upload a wrong format, a message will appear: "*Your shop name* says Cannot upload file".
+When an image is uploaded, it will appear in the logo field on the left, as well as in every panel of the Carrier Wizard.
+
+ - Tracking URL. Has an explanation below the field: "For example: 'http://example.com/track.php?num=@' with '@' where the tracking number should appear.". 
+A tooltip is displayed: "Delivery tracking URL: Type '@' where the tracking number should appear. It will be automatically replaced by the tracking number." The URL should be as displayed in the explanation below, if it's not entered by the explanation, and trying to save will show the message: "The url field is invalid.".
```

Comment:
```suggestion
A tooltip is displayed: "Delivery tracking URL: Type '@' where the tracking number should appear. It will be automatically replaced by the tracking number." The URL should be written as displayed in the tooltips, otherwise, the following error message is displayed: "The url field is invalid.".
```